.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "testdata/lib/forge-std"]
+	path = testdata/lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
```

Comment:
Chisel includes `Vm.sol` in the binary for use within the compilation source, so the inclusion of cheatcodes would have to be reworked if we decide to remove it. The idea here was to not have to fetch the source from GitHub so that chisel can be used in an offline setting.

See: https://github.com/whitenois3/foundry/blob/feat/repl/chisel/src/session_source.rs#L20